src/devtools/client/debugger/src/actions/breakpoints/index.js

```diff
@@ -151,6 +152,7 @@ export function addBreakpointAtLine(cx, line, shouldLog = false, disabled = fals
     if (!source) {
```

Comment:
[Outline](https://github.com/RecordReplay/devtools/blob/master/src/devtools/client/debugger/src/components/PrimaryPanes/Outline.js#L322) component uses [`getSelectedSourceWithContent`](https://github.com/RecordReplay/devtools/blob/master/src/devtools/client/debugger/src/reducers/sources.js#L544) to retrieve the selected source. I wonder should we use the same thing instead of [`getSelectedSource`](https://github.com/RecordReplay/devtools/blob/master/src/devtools/client/debugger/src/reducers/sources.js#L532)?

Reply:
i am not sure if we need the source content here.